src/main/java/com/salam/task_manager/mapper/TaskMapperImplement.java

```diff
@@ -0,0 +1,54 @@
+package com.salam.task_manager.mapper;
+
+import com.salam.task_manager.dto.TaskDto;
+import com.salam.task_manager.exception.ResourceNotFoundException;
+import com.salam.task_manager.models.TaskModel;
+import com.salam.task_manager.models.user.User;
+import com.salam.task_manager.repository.UserRepository;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TaskMapperImplement implements TaskMapper{
```

Comment:
You don't need to implement this

Reply:
Yes, the `@Mapper(componentModel = "spring") ` in TaskMapper interface was generating some source but somehow it does not fill the userId when converting to dto so I choose to handle this myself.

Reply:
[https://github.com/darthmal/Task-Manager-Spring-boot-project/pull/1/commits/94b7376d5caddc1942873ecb8577f5e20a2d3ce6](url)